CHANGELOG.md

```diff
@@ -12,6 +12,58 @@ rely on parsing the output from multiple nodes should pass the `--log-dir` flag
 to `tsh ssh`, which will create a directory where the separated output of each node
 will be written.
 
+#### Container Images
+
+Teleport 15 contains several breaking changes to improve the default
+security and usability of container images.
+
+##### "Fat" container images are discontinued
+
+In order to increase default security in 15+, Teleport will no longer publish
+[container images containing a shell and rich command line environment](https://github.com/gravitational/teleport/blob/branch/v14/build.assets/charts/Dockerfile)
+to https://gallery.ecr.aws/gravitational/teleport. Instead, all users should
+use the [distroless images](https://github.com/gravitational/teleport/blob/branch/v15/build.assets/charts/Dockerfile-distroless)
+introduced in Teleport 12. These images can be found at:
+
+* https://gallery.ecr.aws/gravitational/teleport-distroless
+* https://gallery.ecr.aws/gravitational/teleport-ent-distroless
+
+For users who need a shell in a Teleport container, a "debug" image is
+available which contains BusyBox, including a shell and many CLI tools. Find the debug images at:
+
+* https://gallery.ecr.aws/gravitational/teleport-distroless-debug
+* https://gallery.ecr.aws/gravitational/teleport-ent-distroless-debug
+
+Do not run debug container images in production environments.
+
+Fat container images will continue to be published for Telport 13 and 14
+throughout the remainder of these release's lifecycle.
+
+##### Multi-architecture Teleport Operator Images
+
+Teleport Operator container images will no longer be published with architecture
+suffixes in their tags (for example: `14.2.1-amd64` and `14.2.1-arm`). Instead,
+only a single tag will be published with multi-platform support (e.g. 15.0.0)
+If you use Teleport Operator images with a version suffix, remove the suffix
+and your client should automatically pull the platform appropriate image.
+Individual architectures may be pulled with the `docker pull --platform <arch>`.
+
+##### Quay.io Registry
+
+The quay.io container registry was deprecated and Teleport 12 is the last
+version to publish images to quay.io. With Teleport 15's release, 12 is no
+longer supported and no new images will be published to quay.io.
+
+For supported versions of Teleport, please find the distroless container images
+in [Teleport's public ECR registry](https://gallery.ecr.aws/gravitational).
+
+Following brownouts in May and June, Teleport will disable pulls from
+quay.io repos on Wednesday, 2024-07-03.
+
+Users who wish to continue to use Teleport v12 and earlier beyond July 2024
+will need to download the unsupported quay.io images they depend on and mirror
```

Comment:
Or they could just switch to the ECR images which will still be there.

Reply:
I see. I was only thinking about v12 users and the time it would take them to transition to a supported version. I mean, who uses an unsupported version of a security product? I guess it's a convenience product too for some and the security is an added benefit.

Should we copy all the old images into ECR? Maintain them for the historical record when in 50 years when they do a documentary on how Teleport took over the world, they can go back to the early versions and still run them?